modules/ROOT/pages/single-sign-on-with-oidc.adoc

```diff
@@ -150,50 +151,66 @@ The expected format for Keys and certificates is PEM (with or without the commen
+ ** `credentials` you can set it with an object containing either a `secret`,`jwt` or `secret-jwt` attribute if Bonita client is configured with a confidential access type in the OIDC provider (`public-client` is set to `false`). See https://www.keycloak.org/docs/latest/securing_apps/#_client_authentication_adapter[Keycloak's client authentication documentation] for more information.
  ** Some additional properties no present in the default file can be useful:
-  *** `credentials` you can set it to `password` or `jwt` if Bonita client is configured with a 'Confidential' access type in the OIDC provider.
   *** `connection-pool-size` for the pool of connection to the OIDC provider server (20 by default)
   *** `truststore` path to a truststore file for OIDC provider servers requiring HTTPS. It should contain one or more trusted host certificates or certificate authorities. You can create this truststore by extracting the public certificate of the OIDC server's SSL keystore. This property is mandatory if you OIDC provider requires HTTPS unless you set `disable-trust-manager` property to `true`. If the truststore requires a password, you also need to add the property `truststore-password`.
   *** `client-keystore` path to a keystore file containing client certificate for two-way SSL when the OIDC client adapter makes HTTPS requests to the OIDC provider server. You also need to add the properties `client-keystore-password` (password for the client keystore) and  `client-key-password` (password for the client's key).
   *** `token-store` indicates where the account information of the logged in user is stored on the client (Bonita bundle). By default it is `session` (HTTP session). `cookie` is also supported by the client adapter but it is not recommended to change it as all Bonita OIDC integration has been validated with the `session` setting.
-  *** `verify-token-audience` can be set to true to verify that the bearer token audience contains the client name (`resource ` property).
+  *** `verify-token-audience` can be set to `true` to verify that the JWT tokens audience (`aud` claim) contains the client Id (`resource` property).
+  *** `redirect-rewrite-rules` can be used to modify the path section of the redirect URI sent to the OIDC provider with the authentication or code exchange requests. By default the URI sent is the request URI as seen by Bonita OIDC module, but it may not be the URI the OIDC provider needs to redirect to if Bonita is behind a reverse proxy for example. The value of this property is an object where each attribute key is a regular expression to which the Redirect URI path is to be matched and the attribute value is the replacement String. The other parts of the URI like the protocol, domain, port or query string cannot be modified. Here is an example:
```

Comment:
"but it may not be the URI the OIDC provider needs to redirect to if Bonita is behind a reverse proxy for example." -> I don't understand what this sentence means.

Reply:
clarified